### DISCUSSION TITLE
Add People to Organization Repository

### DISCUSSION THREAD

**Discussion by rience** [community/community#21970] [New to GitHub]
<p>Hi All,</p>
<p>When I want to add member of my organisation to repository - I’d like to be able to search members using their first or last name (I don’t know their GitHub nicknames - have to look them up every time).</p>
<p>But when I go to “Collaborators and Teams” and I click “Add People”</p>
<ul>
<li>When I type in their first name - I’m getting all people from around the world</li>
<li>I need to know their nick name</li>
</ul>
<p>Wouldn’t it make sense to first show results from your organisation and also allow to search by their firstname?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hello - and thank you for this feedback!</p>
<p>Please submit your feedback via the new GitHub <a href="https://github.com/github/feedback/discussions">Feedback Discussions forum</a> so that our product team can better track your request :slight_smile:. It’s a new thing, but we’re trying to push all product feedback to the same new area so it’s easier to find and consolidate user feedback!</p>
